x-pack/elastic-agent/pkg/agent/cmd/enroll_cmd.go

```diff
@@ -7,9 +7,9 @@ package cmd
 import (
 	"bytes"
 	"context"
+	stderror "errors"
```

Comment:
[Naming] `stderror` gave me `os.Stderr` vibes when I was reading the code... can we use a different alias like `goerrors` or `stdliberrors` maybe ?

Even better: we could get rid of the `"github.com/elastic/beats/v7/x-pack/elastic-agent/pkg/agent/errors"` import and only use standard library `errors` package